### DISCUSSION TITLE
[Bug]: integrating  a cascade of 2 inverter

### DISCUSSION THREAD

**Discussion by pmiola73** [wlcrs/huawei_solar#362] [Setting up the integration]
### Describe the issue

Hi I have 2 inverter in daisy chain, master has IDs 16, slave has Ids 1 + 1 dongle and 1 battery connected to every inverter

I try to configure the integration using this string in the slave ids field 16,1 I get an error.

any ideas?

### Bescribe your Huawei Solar Setup

Inverter Type: SUN2000-6KTL-M1
Inverter Firmware version: V100R001C00SPC153
SDongle present: yes 
Power meter present: three phase
Battery:  LUNA2000 10KWh
Battery Firmware version: V100R002C00SPC119

### How do you connect to the inverter?

Via the SDongle, wired connection

### Upload your Diagnostics File

Drag & Drop your Diagnostics File here.
![Screenshot 2023-05-02 at 16 39 53](https://user-images.githubusercontent.com/63242478/235700187-5c1e13f6-3573-4b55-99da-da0c4ad5b0ba.png)

### Upload your relevant debug logs

```text
Logger: backoff
Source: custom_components/huawei_solar/config_flow.py:139
Integration: Huawei Solar
First occurred: 17:04:03 (1 occurrences)
Last logged: 17:04:03

Giving up _do_read(...) after 6 tries (asyncio.exceptions.TimeoutError)



Logger: custom_components.huawei_solar.config_flow
Source: custom_components/huawei_solar/config_flow.py:384
Integration: Huawei Solar
First occurred: 17:04:03 (1 occurrences)
Last logged: 17:04:03

Could not connect to slave 1
```

### Please confirm the following:

- [X] The problem is still present in the latest release of this integration.
- [ ] I did not find an existing issue describing this problem.
- [ ] I did upload the diagnostics-file that I could retrieve from  the 'Devices & Services Page'
- [ ] I did upload the relevant debug logs (via 'Enable Debug Logging'-feature or by manually configuring HA logging)

**Comment by pmiola73** [comment]
this is the log file with debug mode and integration package version 1.2.6b5

thanks

[home-assistant_2023-05-03T11-00-43.419Z.log](https://github.com/wlcrs/huawei_solar/files/11381821/home-assistant_2023-05-03T11-00-43.419Z.log)

**Comment by dettofatto** [comment]
Did you resolve?
